docs/reference/api/usb/USBAudio.md

```diff
@@ -0,0 +1,35 @@
+## USBAudio
+
+<span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_audio.png)<span>USBAudio class hierarchy</span></span>
+
+You can use the USBAudio interface to send and receive audio across USB. By selecting a USBAudio device as your PC's speaker output, you can send audio data. You can also use it as your PC's microphone and receive sound sent by USBAudio.
```

Comment:
What are some best practices for using USBAudio? What are some concepts you need to understand to use it?